src/goto-programs/rebuild_goto_start_function.h

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************\
+ Module: C Test Case Generator
+ Author: Thomas Kiley, thomas@diffblue.com
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_PROGRAMS_REBUILD_GOTO_START_FUNCTION_H
+#define CPROVER_GOTO_PROGRAMS_REBUILD_GOTO_START_FUNCTION_H
+
+#include <util/message.h>
+#include <util/symbol_table.h>
```

Comment:
@thk123 I don't think I have access to that discussion, so I'd appreciate if you could email it to me or share in some other way. My views in this area is largely guided by "Large-scale C++ Software Design" by John Lakos. Whichever way the decision goes: yes, this should go in the coding standards.